db/schema.rb

```diff
@@ -307,14 +308,15 @@
     t.datetime "deleted_at"
     t.datetime "created_at",                            null: false
     t.datetime "updated_at",                            null: false
+    t.string   "guid_ref"
     t.index ["commodity_category_id"], name: "index_dispatch_items_on_commodity_category_id", using: :btree
     t.index ["commodity_id"], name: "index_dispatch_items_on_commodity_id", using: :btree
     t.index ["dispatch_id"], name: "index_dispatch_items_on_dispatch_id", using: :btree
     t.index ["project_id"], name: "index_dispatch_items_on_project_id", using: :btree
   end
 
   create_table "dispatches", force: :cascade do |t|
```

Comment:
@getbez this issue (CATS-1633) is for receipt (GRN) but we're updating dispatches which is part of CATS-1634.